src/main/java/com/terraformersmc/modmenu/gui/widget/UpdateAvailableBadge.java

```diff
@@ -2,12 +2,11 @@
 
 import com.mojang.blaze3d.systems.RenderSystem;
 import net.minecraft.client.gui.DrawContext;
-import net.minecraft.client.util.math.MatrixStack;
 import net.minecraft.util.Identifier;
 import net.minecraft.util.Util;
 
 public class UpdateAvailableBadge {
-	private static final Identifier UPDATE_ICON = new Identifier("icon/trial_available");
+	private static final Identifier UPDATE_ICON = Identifier.of("icon/trial_available");
```

Comment:
Could the `Identifier.ofVanilla` factory method be used instead? The `Identifier.of` method with a single parameter will try to parse `:` from a string, which isn't necessary in this case.

```suggestion
	private static final Identifier UPDATE_ICON = Identifier.ofVanilla("icon/trial_available");
```

Reply:
I was loathe to use the `ofVanilla` because the fabric team changed this Yarn naming a few times now.